### PR TITLE
Add Nord theme

### PR DESCRIPTION
**Pull request by ikalnytskyi**
Basic support for Nord theme.

vim-clap-nord-0
vim-clap-nord-1

### CODE REVIEW

**Comment by xulongwu4** [comment]
This looks great. BTW, there is a discussion about adding support for `vim-clap` in `nord-vim` here: https://github.com/arcticicestudio/nord-vim/pull/178

File: autoload/clap/themes/nord.vim [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+let s:nord13_term = '3'
+let s:nord14_term = '2'
+let s:nord15_term = '5'
+
+let s:palette = {}
+let s:palette.display = {
+  \ 'guibg': s:nord1_gui,
+  \ 'ctermbg': s:nord1_term,
+  \ 'guifg': s:nord4_gui,
+  \ 'ctermfg': s:nord5_term
```
Comment (xulongwu4) [inline autoload/clap/themes/nord.vim:44]: A minor suggestion: the `cterm` color corresponding to `s:nord4_gui` should be `NONE`? That way, `vim` will use the default foreground color for this highlight group.

Comment (ikalnytskyi) [reply autoload/clap/themes/nord.vim:44]: Thanks, good point. I'll update PR.

**Comment by ikalnytskyi** [comment]
> This looks great. BTW, there is a discussion about adding support for vim-clap in nord-vim here: arcticicestudio/nord-vim#178

Yeah, I saw it. I don't know how long it will take to have it merged. The PR has been submitted last year and is not merged yet. I figured that we (and I mean Nord users) may have more lack trying to deliver Nord support to vim-clap. :)

**Comment by xulongwu4** [comment]
@ikalnytskyi One more question: can you tell me how you set the highlight color for the devicons? Thanks.

**Comment by ikalnytskyi** [comment]
@xulongwu4 Honestly, I don't know. I didn't do anything special. I just set the same rules for current selection and current selection sign and nothing except that.

File: autoload/clap/themes/nord.vim [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  \ s:palette.selected,
+  \ 'keep'
+\ )
+let s:palette.current_selection_sign = s:palette.current_selection
+let s:palette.preview = {
+  \ 'guibg': s:nord2_gui,
+  \ 'ctermbg': s:nord3_term
+\ }
+
+let g:clap#themes#material_design_dark#palette = s:palette
```
Comment (liuchengxu) [inline autoload/clap/themes/nord.vim:79]:
```suggestion
let g:clap#themes#nord#palette = s:palette
```

**Comment by liuchengxu** [comment]
Merged, thanks!

**Pull request merged**